webroot/src/components/Lists/Pagination/Pagination.vue

```diff
@@ -11,8 +11,8 @@
             <li
                 v-for="{id, clickable, selected, displayValue} in pages"
                 :key="id"
-                @click="clickable ? setPage(id) : null"
-                @keyup.enter="clickable ? setPage(id) : null"
+                @click="clickable ? setPage(id, NaN) : null"
+                @keyup.enter="clickable ? setPage(id, NaN) : null"
```

Comment:
Hmm, why `NaN`? I wonder if this might be more readable passing in an object to `setPage`. Thoughts?

Reply:
I was struggling with this one and landed on NaN because there is quirkyness with cleanly knowing the page diff. In a scenario where we know all the paging metadata from the server for each request, I wouldn't have needed this. I landed on NaN because A) there is uncertainty / state-quirkyness with having to fetch beyond the last page to know what the last page is, and B) it allowed me to isolate that uncertainty without having to upend the core fundamentals of our current paging handling.

For 2 params, I don't know that I feel strongly about changing to a config object. The `setPage()` signature feels like it's defined ok in the `.ts` file. Let me know how all of that sounds to you.

Reply:
In a quick reading of the code, it doesn't seem like `undefined` would behave any differently. Or since there is a default, did you just want to avoid writing `setPage(id, undefined)`?

Reply:
Yep, pretty much trying to avoid `setPage(id, undefined)` at that point. If you prefer `undefined` I don't mind changing it, just let me know.

Reply:
alternatively, why not just remove the default on `increment` and do `setPage(id)`? I think that's probably my preference unless there's an issue with that I don't see. This is a nit, I realize, but I think a new developer might find `NaN` confusing?

If my understanding is correct here, the values are either -1, 1, or some other value (to indicate an arbitrary page change. Let me know if that's not right, though?

Reply:
I like that idea. Seemed to implement fine 👍